apps/zoom/app/api/auth/refreshAccessToken/route.ts

```diff
@@ -0,0 +1,99 @@
+import { getZoomCredentials } from "@/lib/auth";
+import { sql } from "@vercel/postgres";
+import axios, { AxiosError } from "axios";
+import { redirect } from "next/navigation";
+import { NextRequest, NextResponse } from "next/server";
+
+type AccessTokenResponseType = {
+  access_token: string;
+  token_type: string;
+  refresh_token: string;
+  expires_in: number;
+  scope: string;
+};
+
+export async function POST(req: NextRequest, res: NextResponse) {
+  // for refreshing access token, fetching data from body
+  const bodyData = await req.json();
+
+  const refresh_token = bodyData.refresh_token;
+  const organization_id = bodyData.organization_id;
+
+  if (!refresh_token || refresh_token.length === 0) {
+    return NextResponse.json("refresh_token not found in the query", {
+      status: 400,
+    });
+  }
+
+  if (!organization_id || organization_id.length === 0) {
+    return NextResponse.json("organization_id not found in the query", {
+      status: 400,
+    });
+  }
+
+  // create table if not exist
+  try {
+    await sql`
+      CREATE TABLE IF NOT EXISTS
+        zoom_credentials ( 
+          id SERIAL PRIMARY KEY, 
+          organization_id VARCHAR(1000) NOT NULL, 
+          access_token VARCHAR(5000) NOT NULL, 
+          refresh_token VARCHAR(5000) NOT NULL, 
+          expires_in INT NOT NULL,
+          token_type VARCHAR(500) NOT NULL,
+          scope VARCHAR(500) NOT NULL
+        )
+    `;
+  } catch (error) {
+    console.log(error);
+    return NextResponse.json("Error creating table for credentials", {
+      status: 500,
+    });
+  }
+
+  try {
+    // getting user matching the following parameters
+    const getExistingUser = await sql`
+        SELECT * FROM zoom_credentials WHERE organization_id=${organization_id} AND refresh_token=${refresh_token};
+    `;
+    if (getExistingUser && getExistingUser.rows.length > 0) {
+      const base64EncodedBody = Buffer.from(
```

Comment:
```suggestion
      const base64EncodedBearer = Buffer.from(
```
Let's called it by its name: `bearer`.